src/pages/signin/index.js

```diff
@@ -0,0 +1,56 @@
+import React, { Component } from 'react'
+
+import Button from '../../components/atoms/button'
+import InputWithLabel from '../../components/molecules/input-with-label'
+import PasswordField from '../../components/molecules/password-field'
+import classes from '../signin/styles.css'
+import FacebookLoginButton from '../../components/organisms/facebook-login-button'
+
+class signIn extends Component {
+  state = {
+    username: null,
+  }
+
+  onFacebookLogin = (loginStatus, resultObject) => {
+    if (loginStatus === true) {
```

Comment:
you don't need to verify if loginStatus is equal to true, just `if(loginStatus)`